elements/lisk-dpos/src/dpos.ts

```diff
@@ -148,6 +149,28 @@ export class Dpos {
 		return this.rounds.calcRoundStartHeight(activeRounds);
 	}
 
+	public async isStandByDelegate(
+		address: string,
+		height: number,
+		stateStore: StateStore,
+	): Promise<boolean> {
+		const forgersList = await getForgersList(stateStore);
+		const relevantRound = this.rounds.calcRound(height);
+		const foundForgerList = forgersList.find(
+			roundRecord => roundRecord.round === relevantRound,
+		);
+
+		if (!foundForgerList) {
+			return false;
+		}
+
+		const isStandby = foundForgerList.standby.find(
+			standByDelegate => getAddressFromPublicKey(standByDelegate) === address,
```

Comment:
this is saved as address, so you can compare directly

Reply:
I was confused by the `generateDelegateLists` utils, that adds a publicKey in the forgers list

Reply:
I've updated `generateDelegateLists` function as well in `utils`